### PR TITLE
events

### PR DESCRIPTION
**Pull request by jeesupark96**
https://user-images.githubusercontent.com/90468158/145885469-47214ca4-d99c-41bc-ae3c-e528750dc95c.mp4

### CODE REVIEW

File: dom-events/buttons.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+}
+
+var mouseEvent = document.querySelector('.click-button');
+
+mouseEvent.addEventListener('click', handleClick, Boolean);
```
Comment (BrettAlbright) [inline dom-events/buttons.js]: so when it says you can pass a `Boolean` value to it, it is talking about a `true` or `false` value. That is related to capturing and is not used often, so most of the time you can omit the third argument.

File: dom-events/buttons.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  console.log(event);
+  console.log(event.target);
+}
+
+var mouseHover = document.querySelector('.hover-button', Boolean);
+mouseHover.addEventListener('mouseover', handleMouseover);
+
+function handleDoubleClick(event) {
+  console.log('button double clicked');
+  console.log(event);
```
Comment (BrettAlbright) [inline dom-events/buttons.js]: don't forget to add labels to all of the console logs!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small changes!

File: dom-events/buttons.js (hunk 1 above)
Comment (jeesupark96) [reply dom-events/buttons.js]: makes sense!

File: dom-events/buttons.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+mouseHover.addEventListener('mouseover', handleMouseover);
+
+function handleDoubleClick(event) {
+  console.log('button double clicked');
+  console.log('event: ', event);
+  console.log('event.target: ', event.target);
+}
+
+var doubleClick = document.querySelector('.double-click-button');
+doubleClick.addEventListener('dblclick', handleDoubleClick, Boolean);
```
Comment (BrettAlbright) [inline dom-events/buttons.js:27]: Dont forget this one!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**